stable/appmesh-inject/templates/webhook.yaml

```diff
@@ -1,6 +1,10 @@
 {{ $tls := fromYaml ( include "appmesh-inject.gen-certs" . ) }}
 ---
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" }}
```

Comment:
This webhook doesn't have the sideEffects field defined, so v1 will not work.
Besides, this chart is deprecated, and the functionality is provided by the new appmesh-controller chart. So lets not update this chart.

Reply:
reverted